app/validators/zip_code_validator.rb

```diff
@@ -2,7 +2,7 @@ class ZipCodeValidator < ActiveModel::EachValidator
   ZIP_CODE_REGEX = /\A\d{5}\z/
 
   def validate_each(record, attr_name, value)
-    unless value =~ ZIP_CODE_REGEX && ZipCodes.has_key?(value)
+    unless value =~ ZIP_CODE_REGEX # && ZipCodes.has_key?(value)
```

Comment:
i think it's okay to just remove this instead of commenting it out. this zip codes yml file dates all the way back to 2020 when we built an honest-to-god in-person vita site matcher where you could enter your zip code and we would find the nearest site based on its coordinates (the reason we have postgis not just postgres). probably was never a great idea to use as a validator long term